test/commons/dom/create-grid.js

```diff
@@ -4,13 +4,18 @@ describe('create-grid', function () {
   var fixture;
   var createGrid = axe.commons.dom.createGrid;
   var fixtureSetup = axe.testUtils.fixtureSetup;
+  var gridSize = axe.constants.gridSize;
 
   function findPositions(grid, vNode) {
     var positions = [];
+
     grid.cells.forEach(function (rowCells, rowIndex) {
```

Comment:
This can now be `grid.loopGridPosition(grid.boundries, ())`. However, we'll need the row/col position of the item and not just the `index`, so the `loopGridPosition` function should be updated to pass that back.